scripts/release/validatepackages.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+const { existsSync, readdirSync } = require( 'fs' );
+const { execSync } = require( 'child_process' );
+const { join } = require( 'upath' );
+const { red, green } = require( 'chalk' );
+
+const releaseDirectory = join( process.cwd(), 'release' );
+
+if ( !existsSync( releaseDirectory ) ) {
+	console.log( red( 'The "release" directory does not exist.' ) );
+
+	process.exit( 1 );
+}
+
+let isSuccess = true;
+
+for ( const directoryName of readdirSync( releaseDirectory ) ) {
+	const path = join( releaseDirectory, directoryName );
+
+	try {
+		execSync( `yarn run publint ${ path } --level=warning --strict`, {
```

Comment:
```suggestion
		execSync( `yarn publint ${ path } --level=warning --strict`, {
```

Cosmetic change. Perhaps I am the only one dev that reads things this way, but:

* `yarn run [command]` points (me) to execute a script specified in a `package.json`,
* `yarn [command]` points to execute a binary from the `node_modules`.